src/Inventory/Regions/Repositories/Region.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+namespace Inventory\Regions\Repositories;
+
+use Inventory\Regions\Models\Regions as RegionModel;
+
+class Region
```

Comment:
@FredPeal namespace on the repository 

RegionReository 

we only leave no suffix name or model and DTOs